modules/AlphaWalletNotifications/.gitignore

```diff
@@ -0,0 +1,98 @@
+# Xcode
```

Comment:
I think we don't need the `.gitignore` file right? Maybe a remnant from when we (wanted?) to have a separate repo for each? Shall we remove them separately?

```
./modules/AlphaWalletCore/.gitignore
./modules/AlphaWalletTrackAPICalls/.gitignore
./modules/AlphaWalletAddress/.gitignore
./modules/AlphaWalletLogger/.gitignore
```

Reply:
don't know, will remove `.gitignore` related to this pr, the rest in another pr